### ISSUE TITLE
Getting InvalidArgumentException when creating a new composer project inside container

### ISSUE THREAD

**Issue by mareksuscak** [wachterjohannes/sulu-docker#30]
The reason being that the target folder is not empty.

![image](https://user-images.githubusercontent.com/613647/51995695-7a3bdc00-24b3-11e9-8b9f-bac2c521d6fb.png)

**Comment by wachterjohannes** [comment]
hmm which file do you have inside this folder?

**Comment by mareksuscak** [comment]
A mapped `/var` folder as specified in [docker-compose.yml](https://github.com/wachterjohannes/sulu-docker/blob/master/docker-compose.yml#L26).

I saw no benefit in mapping some of the volumes (such as mysql data volume) to the project folder so I changed it. I also saw no benefit in mapping logs since Sulu spits them out into the project folder already and I don't need to access mysql or elasticsearch logs. I can always see the output of the container itself, should I need to do so.
